### PR TITLE
Bruce/modal payment mobile css fixes

### PR DESCRIPTION
**Pull request by bvillaroman**

### CODE REVIEW

File: src/components/ModalPayment/SquareModal.tsx [hunk 1]
```diff
@@ -150,69 +139,71 @@ const ModalPayment = ({
...
+            </div>
+            {purchaseTypePhrase === 'Donation' ? (
+              <p>
+                By proceeding with your transaction, you understand that you are
+                making a donation to {sellerName}. No goods or services were
+                exchanged for this donation.
+              </p>
+            ) : (
+              <p>
+                By proceeding with your purchase, you understand that the gift
```
Comment (jtmckibb) [inline src/components/ModalPayment/SquareModal.tsx:188]: This copy  should say voucher

File: src/components/ModalPayment/ModalPayment.tsx [hunk 2]
```diff
@@ -26,44 +26,39 @@ const ModalPayment = ({
...
+      seller_id: sellerId,
+    };
+    const buyer: Buyer = { name, email };
     // returns stripe payment intent
     await makePayment(stripe, elements, payment, buyer);
     showConfirmModal(); // shows confirmation modal box
   };
 
+  const purchaseTypePhrase =
+    purchaseType === 'donation' ? 'Donation' : 'Gift card purchase';
```
Comment (jtmckibb) [inline src/components/ModalPayment/ModalPayment.tsx:55]: This should be "Voucher purchase"

File: src/components/ModalPayment/ModalPayment.tsx [hunk 3]
```diff
@@ -26,44 +26,39 @@ const ModalPayment = ({
...
-  const showConfirmModal = () => setIsShown(true);
   const [isChecked, setChecked] = useState(false);
-  const checkAgreement = () =>
-    isChecked ? setChecked(false) : setChecked(true);
   const [name, setName] = useState('');
   const [email, setEmail] = useState('');
 
-  const buyer: Buyer = { name, email };
-
   const stripe = useStripe();
```
Comment (jtmckibb) [inline src/components/ModalPayment/ModalPayment.tsx:34]: FYI we deleted Stripe in the backend today. We can do the same in the frontend too

Comment (bvillaroman) [reply src/components/ModalPayment/ModalPayment.tsx:34]: if stripe was deleted, then im going to delete the whole integration with stripe (ModalPayment file) and keep Square Modal the way it is

File: src/components/ModalPayment/ModalPayment.tsx (hunk 2 above)
Comment (bvillaroman) [reply src/components/ModalPayment/ModalPayment.tsx:55]: will change in the square modal

File: src/components/ModalPayment/SquareModal.tsx (hunk 1 above)
Comment (bvillaroman) [reply src/components/ModalPayment/SquareModal.tsx:188]: got it

File: src/components/ModalPayment/ModalPayment.tsx (hunk 3 above)
Comment (jtmckibb) [reply src/components/ModalPayment/ModalPayment.tsx:34]: nice

**Pull request merged**